Sources/Filters/Sources/TriangleSource/index.js

```diff
@@ -0,0 +1,81 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkMatrixBuilder from 'vtk.js/Sources/Common/Core/MatrixBuilder';
+
+// ----------------------------------------------------------------------------
+// vtkTriangleSource methods
+// ----------------------------------------------------------------------------
+
+function vtkTriangleSource(publicAPI, model) {
```

Comment:
I think that your implementation is very specific (isosceles triangles). Maybe that can be something more generic. @finetjul , what do you think ?

Reply:
My bad. That's an error. This class seems good.